client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -159,15 +162,6 @@ export class CreditCardPaymentBox extends React.Component {
 		);
 	};
 
-	paymentBoxActions = () => {
-		let content = this.paymentButtons();
```

Comment:
Good improvement! (No need to render the paymentButtons() if we're not going to show them)